.github/CONTRIBUTING.md

```diff
@@ -4,96 +4,97 @@ Thank you for your interest in our implementation of the Polkadot Runtime Enviro
+One important distinction is that we are building the Polkadot Runtime Environment, not Polkadot itself. Given that, although a deep understanding of Polkadot is helpful, it's not critical to contribute to gossamer. To help understand how the Runtime Environment relates to Polkadot, check out this [talk that one of our team members gave at DotCon](https://www.youtube.com/watch?v=nYkbYhM5Yfk).
 
 ## Contribution Steps
 
-**1. Fork the gossamer repo.**
+1. **Fork the gossamer repo.**
+2. **Create a local clone of gossamer.**
 
-**2. Create a local clone of gossamer.**
+    ```sh
+    go get -u github.com/ChainSafe/gossamer
+    cd $GOPATH/src/github.com/ChainSafe/gossamer
+    git init
+    ```
 
-```
-go get -u github.com/ChainSafe/gossamer
-cd $GOPATH/src/github.com/ChainSafe/gossamer
-git init
-```
-You may encounter a `package github.com/ChainSafe/gossamer: no Go files in ...` message when doing `go get`. This is not an error, since there are no go files in the project root.
+    You may encounter a `package github.com/ChainSafe/gossamer: no Go files in ...` message when doing `go get`. This is not an error, since there are no go files in the project root.
 
-**3. Link your local clone to the fork on your Github repo.**
+3. **Link your local clone to the fork on your Github repo.**
 
-```
-$ git remote add your-gossamer-repo https://github.com/<your_github_user_name>/gossamer.git
-```
+    ```sh
+    git remote add your-gossamer-repo https://github.com/<your_github_user_name>/gossamer.git
+    ```
 
-**4. Link your local clone to the ChainSafe Systems repo so that you can easily fetch future changes to the ChainSafe Systems repo.**
-     
-```
-$ git remote add gossamer https://github.com/ChainSafe/gossamer.git
-$ git remote -v (you should see myrepo and gossamer in the list of remotes)
-```
+4. **Link your local clone to the ChainSafe Systems repo so that you can easily fetch future changes to the ChainSafe Systems repo.**
 
-**5. Find something to work on.**
+    ```sh
+    git remote add gossamer https://github.com/ChainSafe/gossamer.git
+    git remote -v (you should see myrepo and gossamer in the list of remotes)
+    ```
 
-To start, check out our open issues. We recommend starting with an [issue labeled `Good First Issue`](https://github.com/ChainSafe/gossamer/issues?q=is%3Aopen+is%3Aissue+label%3A%22Good+First+Issue%22). Leave a comment to let us know that you would like to work on it. 
+5. **Find something to work on.**
 
-Another option is to improve gossamer where you see fit based on your evaluation of our code. In order to best facilitate collaboration, please create an issue before you start working on it.
+    To start, check out our open issues. We recommend starting with an [issue labeled `Good First Issue`](https://github.com/ChainSafe/gossamer/issues?q=is%3Aopen+is%3Aissue+label%3A%22Good+First+Issue%22). Leave a comment to let us know that you would like to work on.
```

Comment:
```suggestion
    To start, check out our open issues. We recommend starting with an [issue labeled `Good First Issue`](https://github.com/ChainSafe/gossamer/issues?q=is%3Aopen+is%3Aissue+label%3A%22Good+First+Issue%22). Leave a comment to let us know that you would like to work on it.
```

Reply:
Ah good 👀  sorry about that, I thought it was a `what` instead of `that` in that sentence 😅